karapace/schema_reader.py

```diff
@@ -8,18 +8,25 @@
 from kafka.admin import KafkaAdminClient, NewTopic
 from kafka.errors import NoBrokersAvailable, NodeNotReadyError, TopicAlreadyExistsError
 from karapace import constants
+from karapace.config import Config
+from karapace.master_coordinator import MasterCoordinator
 from karapace.schema_models import SchemaType, TypedSchema
 from karapace.statsd import StatsClient
 from karapace.utils import KarapaceKafkaClient
 from threading import Event, Lock, Thread
-from typing import Dict
+from typing import Any, Dict, Optional
 
 import logging
 import time
 import ujson
 
 log = logging.getLogger(__name__)
 Offset = int
+Subject = str
+Version = int
+Schema = Dict[str, Any]
+# Container type for a subject, with configuration settings and all the schemas
+SubjectData = Dict[str, Any]
```

Comment:
I tried making this into `dataclass`, but this data is read by the API and it resulted in lots of changes.

I also considered using a `TypedDict` here, but that is not available for 2.7 (which is a supported version). It could be done in a follow up if we decide to either remove 2.7 support or to include the `typing-extensions` dependency.